services/web/client/source/class/osparc/utils/Utils.js

```diff
@@ -268,7 +268,7 @@ qx.Class.define("osparc.utils.Utils", {
       const product = qx.core.Environment.get("product.name");
       switch (product) {
         case "s4l":
-          logosPath = lightLogo ? "osparc/s4l_zmt-white.png" : "osparc/s4l_zmt-black.png";
+          logosPath = lightLogo ? "osparc/s4l_zmt-white.svg" : "osparc/s4l_zmt-black.svg";
```

Comment:
The s4l logo font has something that disturbs my need of symmetry. :grin:

Reply:
s4l logo part is perfectly symmetric nor 👌